src/main/java/org/folio/service/titles/TitlesInstanceService.java

```diff
@@ -0,0 +1,43 @@
+package org.folio.service.titles;
+
+import org.folio.rest.core.models.RequestContext;
+import org.folio.rest.jaxrs.model.Title;
+import org.folio.rest.tools.utils.TenantTool;
+import org.folio.service.inventory.InventoryInstanceManager;
+
+import io.vertx.core.Future;
+
+public class TitlesInstanceService {
```

Comment:
propose to rename to TitleInstanceService, it is one to one association

Reply:
Renamed